### PR TITLE
Add jennys FAQ and Events changes

### PR DESCRIPTION
**Pull request by Ciuffi**
Releaseing these CMS changes to prod

### CODE REVIEW

**Review by panjenny0 — APPROVED** [review]
:shipit:

**Comment by dchen150** [comment]
i'm not 100% sure if this is the right place to be putting this but i noticed FAQ and LiveSite don't have that dark background when selected like the websites do - is that intentional?

image
image

**Comment by ianmah** [comment]
@dchen150 intentional as in I purposely avoided it because I didn't feel like fixing, but not necc intentional as a design choice

**Comment by Ciuffi** [comment]
@dchen150 @ianmah fair enough tbh.. Although I actually kind of like it as a design choice as it shows the difference between selected super menus and sub menus.

**Comment by yungalyx** [comment]
lgtm!

File: utility/firebase.js [hunk 1]
```diff
@@ -43,17 +43,26 @@ const faqCollection = FAQ;
...
 const CMSCollection = 'CMS';
+const LivesiteCollection = 'Livesite';
 
-export const formatDate = (date) => {
+export const formatDate = (date, preProcessed = false) => {
   if (!date) {
     return 'invalid date';
   }
-  date = new Date(date * 1000);
   const timeZoneOffset = new Date().getTimezoneOffset() * 60000;
```
Comment (yungalyx) [inline utility/firebase.js:52]: huehue no more 3 milleniums issue

**Review by yungalyx — APPROVED** [review]

**Pull request merged**